client/app/cloud/project/billing/rights/cloud-project-billing-rights.html

```diff
@@ -21,8 +21,8 @@
                             name="ownerContactInput"
                             type="text"
                             class="form-control ng-pristine ng-untouched ng-valid ng-valid-maxlength"
-                            autofocus
-                            autoselect
+                            cuc-autofocus
```

Comment:
You sure this isn't HTML5? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autofocus

Reply:
@FredericEspiau You're right! Since we have to migrate a large amount of components in order to be able to migrate to the monorepo, we are mainly focused of redefined the header (meaning import statements).

We don't want to introduce any changes to the actual codebase.